src/java/fr/paris/lutece/plugins/workflow/modules/appointmentants/service/TaskAntsAppointmentService.java

```diff
@@ -663,14 +709,86 @@ public static List<AntsStatusResponsePOJO> getStatusResponseAsObject( String res
 		{
 			String fieldName = fieldNames.next( );
 			JsonNode field = jsonNode.get( fieldName );
-			statusList.add(
-					mapper.readerFor( AntsStatusResponsePOJO.class )
-					.readValue( field.toString() )
-					);
+
+			AntsStatusResponsePOJO responsePOJO = mapper.readerFor( AntsStatusResponsePOJO.class )
+					.readValue( field.toString() );
+
+			responsePOJO.setAntsApplicationValue( fieldName );
+
+			statusList.add( responsePOJO );
 		}
 		return statusList;
 	}
 
+	/**
+	 * Compare the data (date, location) of the appointment being processed to the data of the appointment currently
+	 * on the ANTS database. This is useful in some specific cases where we try to delete an ANTS appointment unsuccessfully
+	 * but still receive a 200 status code.
+	 *
+	 * @param idAppointment
+	 * 				ID of the appointment being processed
+	 * @param applicationContent
+	 * 				Map containing the appointment data (date, location, etc.)
+	 * @param antsStatusResponse
+	 * 				The content of the response from the ANTS after a GET STATUS request
+	 * @throws AntsException
+	 */
+	private static void compareLocalAppointmentDataWithAntsData( int idAppointment, Map<String, String> applicationContent, AntsStatusResponsePOJO antsStatusResponse )
```

Comment:
This method was made to allow the logging of additional information, as described on Redmine:

- https://dev.lutece.paris.fr/bugtracker/issues/26858#note-3